src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder reportForPeriod = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo).append("\r\n");
+
+        DateTimeFormatter f = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
If you create a formatter, make it a constant field. This will make your code easier to understand and maintain.